JS/Lesson31/saffet/my-app/src/components/CreateTask/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import './style.css';
+
+export const CreateTask = ({ onCreateProps }) => {
+
+  const [task, setTask] = useState("");
+
+  const handleInputChange = (event) => {
+    setTask(event.target.value);
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(task);
+
+    onCreateProps(task);
+    setTask("");
+  }
+
+  return (
+    <>
+      <h2>Create a task in below</h2>
+      <form className="form" action="">
+        <input onChange={handleInputChange} className="form-input" type="text" />
```

Comment:
I'd recommend to change the input field like this:
```suggestion
       <input value={task} onChange={handleInputChange} className="form-input" type="text" />
```
I passed the `task` to input as `value` so that it can be reset easily once form is submitted. OnChange, the `task` value is updated.